docs/website/content/v0.3/en/components/apid.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'apid'
+---
+
+When interacting with Talos, the gRPC api endpoint you will interact with directly is apid.
+Apid acts as the gateway for all component interactions.
+Apid provides a mechanism to route requests to the appropriate destination.
```

Comment:
In other component docs we use the pre-formatted lowercase version (e.g. `apid`) of the name. To give a consistent feel, we should do that in this doc as well (Except for sentences that start with the service name, "Apid" , like you have it is fine).